### DISCUSSION TITLE
"API rate limit exceeded"

### DISCUSSION THREAD

**Discussion by BilElSicario** [gorilla-devs/ferium#449] [Q&A] [answered]
how do I use the "--github-token" option properly, I have my token but can't use the option properly to set it on ferium

**Comment by theRookieCoder** [comment, marked as answer]
You need to add `--github-token <token>` in the beginning of any commands you issue. I would recommend you use the environment variable instead: `export GITHUB_TOKEN=<token>`. You can put that line in your `.bashrc` or equivalent to persist it.
